ophyd/device.py

```diff
@@ -1079,7 +1079,7 @@ def _summary(self):
         "Return a string summarizing the structure of the Device."
         read_attrs = self.read_attrs
         config_attrs = self.configuration_attrs
-        used_attrs = set(read_attrs + config_attrs)
+        used_attrs = list(read_attrs) + list(config_attrs)
```

Comment:
We lost `set` here to make a unique set of attrs. Was it intentional?

Reply:
Good catch, not sure it was intentional but now I see the value of `set()` here.

Reply:
But, cannot use `+` operation with sets.  Will replace with `and`.